scripts/projection_script.r

```diff
@@ -0,0 +1,65 @@
+setwd("/home/tec0/covid_booster_uptake")
```

Comment:
Remove this line: it's user specific and unnecessary... this should run from any place on your machine!

Reply:
Yep, my mistake. I actually knew this one and just forgot!